lib/modules/platform/gitlab/readme.md

```diff
@@ -2,39 +2,46 @@
 
 ## Authentication
 
-First, [create a Personal Access Token](https://docs.gitlab.com/ee/user/profile/personal_access_tokens.html) for the bot account.
+You can authenticate Renovate to GitLab, with a Personal Access Token, or Group Access Token.
 
-If you are using a Group access token, the token must have Developer role or higher permissions in order to create issues and merge requests.
-The token must have Maintainer permissions in order to perform Automerge.
+To start, create either:
 
-For real runs, give the PAT these scopes:
+- a [Personal Access Token](https://docs.gitlab.com/ee/user/profile/personal_access_tokens.html) for the bot account
+- or a [Group Access Token](https://docs.gitlab.com/ee/user/group/settings/group_access_tokens.html#bot-users-for-groups) for the bot account
+
+The bot account must have at least the Developer role in order to [create issues and merge requests](https://docs.gitlab.com/ee/user/permissions.html#project-members-permissions).
+If you are using automerge, the bot account must have the appropriate ["Allowed to merge" permission on the protected branch](https://docs.gitlab.com/ee/user/project/protected_branches.html#require-everyone-to-submit-merge-requests-for-a-protected-branch) of your projects.
+If merging is restricted to Maintainers, the bot account must have the Maintainer role.
+
+If you are using a group access token, to keep using the same GitLab-generated bot user you must [rotate/refresh the Group Access Token](https://docs.gitlab.com/ee/api/group_access_tokens.html#rotate-a-group-access-token) _before_ the token's expiry date.
```

Comment:
Should we document about ignorePrAuthor if they _don't_ do this?

Reply:
> Should we document about ignorePrAuthor if they _don't_ do this?

Is this what you're referring to?

1. User creates and uses token for `renovate-bot-1`
2. Token expires
3. User creates _new_ token for the same `renovate-bot-1` account
4. User now needs to set `ignorePrAuthor=true` to keep things working?

I'm not sure _why_ you need to set `ignorePrAuthor=true` when you only change the token? It's probably related to renaming-attacks, where people use a trusted name on a bad bot. Probably the token has the ID/authorization for the user's bot?

## `ignorePrAuthor` docs a bit confusing

The `ignorePrAuthor` docs are confusing me:

- One line says: "usecase GitLab users who _rotate tokens_"
- Two lines after says: "Only want to enable if you are _changing_ the bot account by _renaming_ the bot"

## Full quote of `ignorePrAuthor` docs:

> ## ignorePrAuthor
> 
> Set to `true` to fetch the entire list of PRs instead of only those authored by the Renovate user.
> 
> | Name | value |
> | --- | --- |
> | type | boolean |
> | default | false |
> | cli |	--ignore-pr-author| 
> | env |	RENOVATE_IGNORE_PR_AUTHOR
>
> This is usually needed if someone needs to migrate bot accounts, including from the Mend Renovate App to self-hosted.
> **An additional use case is for GitLab users of project or group access tokens who need to rotate them.**
> 
> If `ignorePrAuthor` is configured to true, it means Renovate will fetch the entire list of repository PRs instead of optimizing to fetch only those PRs which it created itself.
> **You should only want to enable this if you are changing the bot account** (e.g. from `@old-bot` to `@new-bot`) and want `@new-bot` to find and update any existing PRs created by `@old-bot`.
> 
> Setting this field to `true` in GitLab will also mean that all Issues will be fetched instead of only those by the bot itself.

Reply:
It could be renamed ignorePlatformAuthor

Reply:
Based on my understanding (from the discussion that lead to this PR) the problem when changing the access token (I.e., a new bot user) is that new dashboard issues and PRs are created since the user does not match anymore. `ignorePrAuthor` solves this. 

It could be helpful to mention this and refer to the relevant docs.

Reply:
If the picture in the user's brain should be closer to "ignore the platform author" than "ignore PR author" a rename seems to `ignorePlatformAuthor` seems good.

> Should we document about ignorePrAuthor if they don't do this?

I'm still confused by this whole thing, so I can only respond in general terms... :upside_down_face: The docs should make it easier for a user to:

- get the right mental picture, or concept of the config option and it's uses
- do the _right_ thing, the _first_ time
- be done reading quickly, and get on with work

If we need to explain better/more to help users, we should do that!